jplag/src/main/java/jplag/options/Options.java

```diff
@@ -181,8 +189,8 @@ public abstract void initializeSecondStep(Program program)
 	public static void usage() {
 		System.out.print(Program.name_long
 						+ ", Copyright (c) 2004-2017 KIT - IPD Tichy, Guido Malpohl, and others.\n"
-						+ "Usage: JPlag [ options ] <root-dir>\n"
-						+ " <root-dir>        The root-directory that contains all submissions\n\n"
+						+ "Usage: JPlag [ options ] <root-dir> [-c file1 file2 ...]\n"

```

Comment:
Having to have a root-dir _and_ the file list seems awkward. When running in the new mode, the file list should be enough.

Reply:
I agree, but it was not very easy to eliminate it quickly, as it required in Submission class. I'll try to fix it this week.

Reply:
It was easier than I expected. Seems that providing a null dir in submission does not break the main functions of jplag. Please, review my latest commit.